heliopy/data/util.py

```diff
@@ -139,6 +139,10 @@ def processing_func(local_dir, local_fname)
 
 
 def units_attach(data, units):
+    unit_key = list(units.keys())
```

Comment:
Instead of assuming unit-less by default, I think we should manually specify unit-less for each variable. This way if someone forgets to add units, instead of incorrectly attaching dimensionless the tests will raise an error.

Reply:
Oh yep, that is a perspective I didn't look at this problem from! But I think in case a user cannot find all the units, there should be an option to fill them with dimensionless quantity or something else! I think there can be a better way to do this. I should remove this and think of something else